src/format.mjs

```diff
@@ -80,3 +88,18 @@ export async function format(configFileLocation, config) {
 
   return newFilePaths;
 }
+
+function convertSourceMapURL(filePath, fileContents) {
```

Comment:
The larger this project is, the more I am considering using TypeScript for authoring it. This might not be the right place, but soon.